oci/annotations.go

```diff
@@ -17,6 +17,8 @@ const (
 	AnnotationKernelKConfig        = "org.unikraft.kernel.kconfig."
 	AnnotationKernelArch           = "org.unikraft.kernel.arch"
 	AnnotationKernelPlat           = "org.unikraft.kernel.plat"
+	AnnotationKernelVolumePath     = "org.unikraft.kernel.volume-%d.path"
+	AnnotationKernelVolumeMount    = "org.unikraft.kernel.volume-%d.mount"
```

Comment:
To be consistent with Unikraft's core:
```diff
- org.unikraft.kernel.volume-%d.path"
+ org.unikraft.kernel.fs%d.path"
- org.unikraft.kernel.volume-%d.mount"
+ org.unikraft.kernel.fs%d.mount"
```

Reply:
I am thinking about the attributes that are coming from the new addition occuring in the core to handle volumes: my/path/fs0:/cool.txt